lib/event_flow/ui/event_flow_section.dart

```diff
@@ -18,18 +18,21 @@ import 'package:flutter/material.dart';
 class EventFlowSection extends StatelessWidget {
   EventFlowSection({
     required this.focusNode,
+    required this.speakerNotifier,
     Key? key,
   }) : super(key: key);
 
   final FocusNode focusNode;
   final sessionRepository = getIt.get<SessionRepository>();
+  final ValueNotifier<Speaker?> speakerNotifier;
```

Comment:
I can add comment but I thought it is self explanatory. The speaker notifier updates its listeners whenever the speaker changes. Since the Speaker extends `Equatable` this guarantees that the listeners will not be triggered if the same speakers with different reference is set as value.

Reply:
So, it is important for the `SpeakerDetail` right, and it will be null again if the dialog is closed, right?

Reply:
@bgoktugozdemir Good point. If the dialog is closed in this line we set it to null back:
https://github.com/flutterturkey/festival_flutterturkiye_org/pull/125/files#diff-5e8bea9462eaf2f9b18a209cc4372326b5f29a44bf4876c20e1ec6c78ebb0bdbR74